benches/generate_data.py

```diff
@@ -2,9 +2,9 @@
 import shapely
 
 
-gdf = gpd.read_file("Utah.geojson.zip", engine="pyogrio")
```

Comment:
This file is from https://github.com/microsoft/USBuildingFootprints#download-links, with a download link (unsure if this is persistent) of https://minedbuildings.z5.web.core.windows.net/legacy/usbuildings-v2/Utah.geojson.zip